### ISSUE TITLE
java.lang.UnsatisfiedLinkError: dlopen failed: library "libmmkv.so" not found

### ISSUE THREAD

**Issue by TheMelody** [Tencent/MMKV#1415]
## Note:
An crash issue will be **ignored & closed** in a week **without logs**.

### The language of MMKV

Kotlin

### The version of MMKV

com.tencent:mmkv:2.0.0

### The platform of MMKV

Android

### The installation of MMKV

gradle

### What's the issue?

几款vivo设备，启动页调用MMKV.initialize(context) ：

机型：V2359A 
机型：V2307A 
机型：V2309A 
机型：V2219A 
机型：V2364A
机型：V2231A 
机型：PA2170 
机型：V2165A 
机型：V1981A 
机型：V2171A 
机型：V2001A 
机型：V2025A 
机型：V1829A 

### What's the log of MMKV when that happened?

错误日志：
```
10-30 20:41:14.856  8958  8958 D nativeloader: Load libmmkv.so using ns clns-7 from class loader (caller=/data/app/~~mGV_A_gUG_9-lluKBOrvmg==/com.myapp.video-b1aL_nAvIgxoxXaqaFg1Jw==/base.apk!classes3.dex): dlopen failed: library "libmmkv.so" not found
10-30 20:41:14.857  8958  8958 D AndroidRuntime: Shutting down VM
10-30 20:41:14.858  8958  8958 E AndroidRuntime: FATAL EXCEPTION: main
10-30 20:41:14.858  8958  8958 E AndroidRuntime: Process: com.myapp.video, PID: 8958
10-30 20:41:14.858  8958  8958 E AndroidRuntime: java.lang.UnsatisfiedLinkError: dlopen failed: library "libmmkv.so" not found
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at java.lang.Runtime.loadLibrary0(Runtime.java:1193)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at java.lang.Runtime.loadLibrary0(Runtime.java:1115)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at java.lang.System.loadLibrary(System.java:1765)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.tencent.mmkv.MMKV.doInitialize(SourceFile:9)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.tencent.mmkv.MMKV.initialize(SourceFile:24)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.tencent.mmkv.MMKV.initialize(SourceFile:3)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at g0.__.d(Unknown Source:33)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at f0._________________._________________________(Unknown Source:42)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.myapp.video.activity.SplashActivity.O(Unknown Source:89)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.myapp.video.activity.SplashActivity.R(Unknown Source:109)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.myapp.video.activity.SplashActivity.F(Unknown Source:34)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at p.g.onClick(Unknown Source:36)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at u.__________________________.onClick(Unknown Source:94)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at androidx.appcompat.app.AlertController$ButtonHandler.handleMessage(SourceFile:38)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:109)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:250)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:340)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:9836)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:621)
10-30 20:41:14.858  8958  8958 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:957)
```

**Comment by TheMelody** [comment]
> MMKV 2.0.0 drops armv7 support. If it's unacceptable to your App, you should stick to the 1.3.x LTS version.

 1.3.x LTS 支持16k pagesize????

**Comment by lingol** [comment]
No. It's the choice you have to make.
https://github.com/Tencent/MMKV/releases/tag/v1.3.7
>Please note that only critical bug fixes will be applied to the 1.3.x series. New features will be introduced in version 2.0 and later, which will discontinue support for 32-bit architectures and raise the minimum SDK version requirement to 23.

**Comment by TheMelody** [comment]
> No, LTS will not add any new features, and the 16 K page size is one of them. You have to make the choice. https://github.com/Tencent/MMKV/releases/tag/v1.3.7
> 
> > Please note that only critical bug fixes will be applied to the 1.3.x series. New features will be introduced in version 2.0 and later, which will discontinue support for 32-bit architectures and raise the minimum SDK version requirement to 23.

溜了溜了，我还是用jetpack dataStore吧

**Comment by lingol** [comment]
> > No, LTS will not add any new features, and the 16 K page size is one of them. You have to make the choice. https://github.com/Tencent/MMKV/releases/tag/v1.3.7
> 
> > 
> 
> > > Please note that only critical bug fixes will be applied to the 1.3.x series. New features will be introduced in version 2.0 and later, which will discontinue support for 32-bit architectures and raise the minimum SDK version requirement to 23.
> 
> 
> 
> 溜了溜了，我还是用jetpack dataStore吧

Sure, suite yourself.

**Comment by TheMelody** [comment]
> > > No, LTS will not add any new features, and the 16 K page size is one of them. You have to make the choice. https://github.com/Tencent/MMKV/releases/tag/v1.3.7
> > 
> > > 
> > 
> > > > Please note that only critical bug fixes will be applied to the 1.3.x series. New features will be introduced in version 2.0 and later, which will discontinue support for 32-bit architectures and raise the minimum SDK version requirement to 23.
> > 
> > 
> > 
> > 溜了溜了，我还是用jetpack dataStore吧
> 
> Sure, suite yourself.

😄😄 洋文很溜啊？

**Comment by lingol** [comment]
16K pagesize is supported in v1.3.14.
